azure-pipelines/steps/publish-code-coverage-results.yml

```diff
@@ -1,7 +1,7 @@
 steps:
-  # This script only runs for once for each host platform (to avoid unnecessary uploads) and will only run when the
-  # variable CODECOV_TOKEN is available, which means this step is disabled for PRs from forked repositories.
-  # Use "bash" rather than "script" to allow upload from all platforms including Windows.
+  # This script only runs for one set of unit tests (to avoid duplicate uploads) and will only run when the variable
+  # CODECOV_TOKEN is available, which means this step is disabled for PRs from forked repositories.
+  # Use "bash" rather than "script" to allow upload from all host platforms including Windows.
```

Comment:
This shouldn't be changed as the description was correct because we upload once for each host platform.

Reply:
😭